src/domain/QuestionLists/MoreIconContent.tsx

```diff
@@ -27,7 +29,9 @@ export default function MoreIconContent({
       </Menu.Item>
       <Menu.Item>
         <Button type="link" danger>
-          Delete
+          <Link onClick={deleteCallback} to="/">
```

Comment:
I think that there is no need to have a Link here, just a button with onClick callback would be sufficient, since we actually do not need to refresh the page after deleting a question set. Unless I am missing something 🤔

Reply:
I kept Link as is above, but you are right it doesn't make sense. :)